oak-run/src/main/java/org/apache/jackrabbit/oak/run/DataStoreCommand.java

```diff
@@ -317,22 +486,27 @@ public VerboseIdLogger(Options options) {
 
             outFileList.add(filterFiles(outDir, "marked-"));
             outFileList.add(filterFiles(outDir, "gccand-"));
+
             outFileList.removeAll(Collections.singleton(null));
 
             if (outFileList.size() == 0) {
                 throw new IllegalArgumentException("No candidate file found");
             }
         }
 
+        static File filterFiles(File outDir, String filePrefix) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/oak-indexing/jackrabbit-oak/pullRequest?prid=5079914)